### ISSUE TITLE
Delayed execution of tags (or anything) in collections module

### ISSUE THREAD

**Issue by spaceChRiS** [darktable-org/darktable#10825]
**Is your feature request related to a problem? Please describe.**

TLDR: It would make sense to have a delayed execution of the current collection that only gets effective when done with the current image(s).

I use to tag people in my photographs manually. In a tagging session with several hundred photos, it would make sense to walk through them and use an exclusion rule to remove those that are done from the list. However, this way I can tag only one person, because the collection is applied immediately.

**Describe the solution you'd like**

It would be great to have an option to execute the collection only after the focus is changed, e.g. to the next image. It should stay an option as both behaviors are useful dependent on the task.

**Alternatives**

One could introduce a special tag that indicates that tagging is done which is added as last tag for each image. However, this would spam the tag list and feels like a cumbersome workaround.

**Additional context**

Due to recent activities regarding the collections, I thought this would be a reasonable point in time to add this feature request, see #10733, ping  @AlicVB :wink:.

**Comment by spaceChRiS** [comment]
Hm, maybe there is also a workflow alternative that I missed … Any idea?

**Comment by spaceChRiS** [comment]
Still interested, therefore I try to keep it alive, I hope this is OK.

**Comment by spaceChRiS** [comment]
Activity …
